proto/cosmwasm/wasm/v1/query.proto

```diff
@@ -65,6 +65,12 @@ service Query {
     option (google.api.http).get = "/cosmwasm/wasm/v1/code";
   }
 
+  // CodeInfo gets the metadata for a single wasm code
+  rpc CodeInfo(QueryCodeInfoRequest) returns (CodeInfoResponse) {
```

Comment:
Should we have a dedicated type here callen `QueryCodeInfoResponse` for consistency? Then we could also use the opportunity to clean it up a little bit